js/adapt-contrib-glossaryItemView.js

```diff
@@ -73,7 +73,7 @@ export default class GlossaryItemView extends Backbone.View {
     });
     $glossaryItemTerm.attr('aria-expanded', true);
     this.model.set('_isDescriptionOpen', true);
-    this.listenToOnce(Adapt, 'drawer:opened', () => this.onAnimationComplete($glossaryItemTerm));
+    this.listenToOnce(Adapt, 'drawer:opened', this.onAnimationComplete($glossaryItemTerm));
```

Comment:
`this.onAnimationComplete($glossaryItemTerm)` executes immediately, suggesting that the event listener isn't needed.
```suggestion
    this.onAnimationComplete($glossaryItemTerm);
```

Reply:
may as well rip out the onAnimationComplete function then too!